### DISCUSSION TITLE
Can we using Laravel Backpack in an offline mode?

### DISCUSSION THREAD

**Discussion by kanmanuso** [Laravel-Backpack/community-forum#1103] [Q&A (Help)]
Dear All,

I am trying to using a Laravel Backpack in the internal environment, without external network connection. I have been using a basset (https://github.com/Laravel-Backpack/basset), but the app is still trying to download external resources from outside (> 100 files).

Has anyone run into the same problem? If yes, then how was it resolved. Thank you in advance.

**Comment by jcastroa87** [comment]
Hello @kanmanuso 

Sure you can, but need to have internet at least one time to allow basset create cache, then you are able to work in offline mode.

When you have internet access try
```
php artisan basset:cache
```

Then when process finish, you will have local copy of assets and will able to work offline.

Cheers.

**Comment by jcastroa87** [comment]
Due to not activity i will close this issue, but please feel free to re-open or create a new one if needed.

Cheers.
